portality/view/doaj.py

```diff
@@ -447,7 +447,7 @@ def support():
 
 @blueprint.route("/support/sponsors/")
```

Comment:
would it be worth changing the route and adding (yet another) legacy redirect? (so long we we've used `url_for('sponsors') everywhere we can change the route without issue)